consensus/misc/eip1559/eip1559.go

```diff
@@ -132,6 +132,52 @@ func ValidateHoloceneExtraData(extra []byte) error {
 	return ValidateHolocene1559Params(extra[1:])
 }
 
+// DecodeMinBaseFeeExtraData decodes the extraData parameters from the encoded form defined here:
+// https://specs.optimism.io/protocol/jovian/exec-engine.html
+//
+// Returns 0,0,0 if the format is invalid, though ValidateMinBaseFeeExtraData should be used instead of this function for
+// validity checking.
+func DecodeMinBaseFeeExtraData(extra []byte) (uint64, uint64, uint64) {
+	// Best effort to decode the extraData for every block in the chain's history,
+	// including blocks before the minimum base fee feature was enabled.
+	if len(extra) == 9 {
+		// This is Holocene extraData
+		denominator, elasticity := DecodeHolocene1559Params(extra[1:9])
+		return denominator, elasticity, 0
+	} else if len(extra) == 17 {
+		// Decode extraData when the minimum base fee fork is enabled
+		denominator, elasticity := DecodeHolocene1559Params(extra[1:9])
+		minBaseFee := binary.BigEndian.Uint64(extra[9:])
+		return denominator, elasticity, minBaseFee
+	}
+	return 0, 0, 0
+}
+
+// EncodeMinBaseFeeExtraData encodes the EIP-1559 and minBaseFee parameters into the header 'ExtraData' format.
+// Will panic if EIP-1559 parameters are outside uint32 range.
+func EncodeMinBaseFeeExtraData(denom, elasticity uint64, minBaseFee uint64) []byte {
```

Comment:
Nit: 
```suggestion
func EncodeMinBaseFeeExtraData(denom, elasticity, minBaseFee uint64) []byte {
```

Also wondering if we just call this `EncodeJovianExtraData`. See https://github.com/ethereum-optimism/design-docs/pull/323

Reply:
[f785e13](https://github.com/ethereum-optimism/op-geth/pull/648/commits/f785e13a24cb3216faba37f6c2e1f5d596cabfe5)